framework/doc/content/source/auxkernels/VariableGradientComponent.md

```diff
@@ -1,31 +1,8 @@
 # VariableGradientComponent
 
-!alert! construction title=Undocumented Class
-The VariableGradientComponent has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
-```markdown
-# VariableGradientComponent
-
-!syntax description /AuxKernels/VariableGradientComponent
-
-## Overview
-
-!! Replace these lines with information regarding the VariableGradientComponent object.
-
 ## Example Input File Syntax
 
-!! Describe and include an example of how to use the VariableGradientComponent object.
-
-!syntax parameters /AuxKernels/VariableGradientComponent
-
-!syntax inputs /AuxKernels/VariableGradientComponent
-
-!syntax children /AuxKernels/VariableGradientComponent
-```
-!alert-end!
+!listing /test/tests/auxkernels/grad_component/grad_component.i block=AuxKernels
```

Comment:
Same here, explain the input file a little.

Reply:
done